src/azure-openapi-validator/tests/composite-azure-test.ts

```diff
@@ -0,0 +1,22 @@
+import { Message } from '../../jsonrpc/types';
+import { suite, test, slow, timeout, skip, only } from "mocha-typescript";
+import { safeLoad } from "js-yaml";
+import { AutoRestPluginHost } from "../../jsonrpc/plugin-host";
+import { run } from "../../azure-openapi-validator";
+import { ReadFileAsString, CollectTestMessagesFromValidator } from "./utilities/tests-helper";
+import * as assert from "assert";
+
+
+@suite class CompositeAzureTest {
+  @test @timeout(120000) async "description should not be parameter name"() {
```

Comment:
I would argue that it would be much harder to determine the tests that are actually hanging up.